contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -982,100 +1021,100 @@ pltsql_post_parse_analyze(ParseState *pstate, Query *query, JumbleState *jstate)
 									def->constraints = lappend(def->constraints, get_rowversion_default_constraint(def->typeName));
 								}
 								break;
-							case AT_AddConstraint:
-								{
-									Constraint *c = castNode(Constraint, cmd->def);
+							case AT_AddConstraint: 
+							{
+								Constraint *c = castNode(Constraint, cmd->def);
 
-									c->conname = construct_unique_index_name(c->conname, atstmt->relation->relname);
+								c->conname = construct_unique_index_name(c->conname, atstmt->relation->relname);
 
-									if (escape_hatch_unique_constraint != EH_IGNORE &&
-										c->contype == CONSTR_UNIQUE &&
-										is_nullable_constraint(c, relid))
-									{
-										ereport(ERROR,
-												(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-												 errmsg("Nullable UNIQUE constraint is not supported. Please use babelfishpg_tsql.escape_hatch_unique_constraint to ignore "
-														"or add a NOT NULL constraint")));
-									}
-
-									if (rowversion_column_name)
-										validate_rowversion_table_constraint(c, rowversion_column_name);
+								if (escape_hatch_unique_constraint != EH_IGNORE &&
+									c->contype == CONSTR_UNIQUE &&
+									is_nullable_constraint(c, relid))
+								{
+									ereport(ERROR,
+											(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+											errmsg("Nullable UNIQUE constraint is not supported. Please use babelfishpg_tsql.escape_hatch_unique_constraint to ignore "
+												"or add a NOT NULL constraint")));
 								}
+
+								if (rowversion_column_name)
+									validate_rowversion_table_constraint(c, rowversion_column_name);
+							}
```

Comment:
Please fix the formatting for rest of the changes. It's not clear what exactly you've modified in this code apart from the formatting.

Reply:
Done